Dockerfile using non-supported JDK

Initially reported from bioinformatics-ua/dicoogle-learning-pack#10. When using Docker to build Dicoogle, the project will be built with the host's JDK, which might not be compatible. The way I see it, the JDK should be properly defined by the Docker image.

With JDK 11 installed on the host system:

```
docker build -t dicoogle .
```

```
[WARNING] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/core/XMLClientSupport.java:[42,26] [deprecation] XMLReaderFactory in org.xml.sax.helpers has been deprecated
[INFO] 16 warnings
[INFO] -------------------------------------------------------------
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR :
[INFO] -------------------------------------------------------------
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[25,32] error: package javax.xml.bind.annotation does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[26,32] error: package javax.xml.bind.annotation does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[32,1] error: cannot find symbol
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/RestEnumField.java:[27,21] error: package javax.xml.bind does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/RestEnumField.java:[28,21] error: package javax.xml.bind does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[34,5] error: cannot find symbol
[INFO] 6 errors
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] Reactor Summary for dicoogle-all 3.0.0-SNAPSHOT:
[INFO]
[INFO] dicoogle-all ....................................... SUCCESS [ 19.004 s]
[INFO] dicoogle-sdk ....................................... SUCCESS [01:15 min]
[INFO] dicoogle-sdk-ext ................................... SUCCESS [  8.635 s]
[INFO] dicoogle ........................................... FAILURE [03:27 min]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:10 min
[INFO] Finished at: 2019-05-23T04:34:10Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:2.3.2:compile (default-compile) on project dicoogle: Compilation failure: Compilation failure:
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[25,32] error: package javax.xml.bind.annotation does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[26,32] error: package javax.xml.bind.annotation does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[32,1] error: cannot find symbol
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/RestEnumField.java:[27,21] error: package javax.xml.bind does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/RestEnumField.java:[28,21] error: package javax.xml.bind does not exist
[ERROR] /root/dicoogle/dicoogle/src/main/java/pt/ua/dicoogle/server/web/rest/elements/JaxbStrList.java:[34,5] error: cannot find symbol
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR]
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <goals> -rf :dicoogle
The command '/bin/sh -c ( cd dicoogle && mvn install && ln -s /root/dicoogle/dicoogle/target/dicoogle.jar /root/ )' returned a non-zero code: 1
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,12 +1,9 @@
-FROM ubuntu
+FROM openjdk:8
 
 WORKDIR /root
 
 RUN apt-get update && \
-    apt-get install apt-utils -y && \
-    apt-get upgrade -y
-
-RUN apt-get install -y software-properties-common build-essential openjdk-8-jre git maven
+    apt-get install -y software-properties-common build-essential git maven
 
 RUN git clone https://github.com/bioinformatics-ua/dicoogle
 RUN ( cd dicoogle && mvn install && ln -s /root/dicoogle/dicoogle/target/dicoogle.jar /root/ )

